bosh/bosh_ssh.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+SCRIPT_DIR=$(cd "$(dirname $_)" && pwd)
+BOSH_PW=${BOSH_PW:-"abc"}
+BOSH=$(which bosh)
+
+if [ -z "${DEPLOYER_CONCOURSE}" ]; then
+  echo "Must provide DEPLOYER_CONCOURSE ip or hostname as \$1 or env. var"
```

Comment:
`DEPLOYER_CONCOURSE` is not read from $1, only as environment variable. this message is misleading